dim/src/scanner/mod.rs

```diff
@@ -41,11 +42,65 @@ use tracing::error;
 use tracing::info;
 use tracing::instrument;
 use tracing::warn;
-use walkdir::WalkDir;
 
 pub use error::Error;
 
-pub(super) static SUPPORTED_EXTS: &[&str] = &["mp4", "mkv", "avi", "webm"];
+pub(super) static SUPPORTED_EXTS: &[&str] = &[
```

Comment:
In testing this feature I realized there were several media files that Dim wasn't picking up so I copied this list from Jellyfin which seems comprehensive, but may also be considered overkill. https://github.com/jellyfin/jellyfin/blob/master/Emby.Naming/Common/NamingOptions.cs#L24-L80

Reply:
little overkill, 100% for it